File: app/src/main/java/com/zzzmode/appopsx/ui/permission/IPermView.java
package com.zzzmode.appopsx.ui.permission;

import com.zzzmode.appopsx.ui.model.OpEntryInfo;
import java.util.List;

/**
 * Created by zl on 2017/5/1.
 */
interface IPermView {

  void showProgress(boolean show);

  void showError(CharSequence text);

  void showPerms(List<OpEntryInfo> opEntryInfos);

  void updateItem(OpEntryInfo info);
}

File: app/src/main/kotlin/com/zzzmode/appopsx/ui/permission/IPermView.kt
package com.zzzmode.appopsx.ui.permission

import com.zzzmode.appopsx.ui.model.OpEntryInfo

/**
 * Created by zl on 2017/5/1.
 */
internal interface IPermView {

    fun showProgress(show: Boolean)

    fun showError(text: CharSequence)

    fun showPerms(opEntryInfos: List<OpEntryInfo>)

    fun updateItem(info: OpEntryInfo)
}
